index.js

```diff
@@ -32,4 +32,29 @@ module.exports = options => options ?
 	getPort(options).catch(() => getPort(Object.assign(options, {port: 0}))) :
 	getPort({port: 0});
 
+module.exports.makeRange = (from, to) => {
+	if (!Number.isInteger(from) || !Number.isInteger(to)) {
+		throw new TypeError('`from` and `to` must ne integer numbers');
+	}
+
+	if (from < 1025 || from > 65535) {
+		throw new RangeError('`from` must be between 1025 and 65535');
+	}
+
+	if (to < 1025 || to > 65536) {
```

Comment:
Why is `to` exclusive?

Reply:
That is like the question I wanted to discuss the most in this PR.
I stick with programming convention of [from,to) ranges, as it's usually default and easier to use when you deal with multiple ranges, but maybe for this project we would want to do inclusive ranges.

Reply:
That is, with conventional ranges
```
intersects(range(a, b), range(b, c)) == false
union(range(a, b), range(b, c)) == range(a, c)
```

Reply:
// CC @sindresorhus